internal/datanode/data_node.go

```diff
@@ -1144,6 +1145,10 @@ func (node *DataNode) AddImportSegment(ctx context.Context, req *datapb.AddImpor
 	// Get the current dml channel position ID, that will be used in segments start positions and end positions.
 	posID, err := ds.getChannelLatestMsgID(context.Background(), req.GetChannelName(), req.GetSegmentId())
 	if err != nil {
+		log.Error("failed to get channel's latest message ID",
```

Comment:
please use log.Warn

Reply:
This is an error that would cause, for example, bulk insert task to fail.